chromium/background.js

```diff
@@ -191,6 +218,8 @@ AppliedRulesets.prototype = {
       this.active_tab_rules[tabId] = {};
       this.active_tab_rules[tabId][ruleset.name] = ruleset;
     }
+
+    updateState();
```

Comment:
I think it makes more sense to place this on what is currently line 325 of this file, so as to eliminate unnecessary calls.